OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -192,6 +228,9 @@ public ActorMap(World world, ActorMapInfo info)
 			actorShouldBeRemoved = removeActorPosition.Contains;
 		}
 
+		/// <summary>
+		/// NOTE: Obsolete. Use GetActorsAt
```

Comment:
Seeing as the interfaces are gone, we should go back to using the original methods.